lib/factory_girl/definition_proxy.rb

```diff
@@ -94,6 +95,16 @@ def transient(&block)
     #
     # are equivalent.
     def method_missing(name, *args, &block)
+      # If methods are called from debugger, original method should be called
+      if ['eval', 'process_commands', 'build_compact_name', 'build_compact_value_attr', 'print_element'].include?(caller_locations[1].base_label) ||
```

Comment:
Line is too long. [148/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.